docs/how-to/notebooks.txt

```diff
@@ -179,18 +179,23 @@ GPUs, set ``resources.slots`` to ``0``:
 To ensure that your work is saved even if your notebook gets terminated,
 it is recommended to launch all notebooks with a shared filesystem
 directory *bind-mounted* into the notebook container and work on files
-inside of the bind mounted directory. For example, a user ``jimmy`` with
-a shared filesystem home directory at ``/shared/home/jimmy`` could use
-the following configuration to launch a notebook:
+inside of the bind mounted directory.
```

Comment:
I think the original documentation for adding a bind mount probably shouldn't be deleted since it is for all the cases.